docs/DEVGUIDE.md

```diff
@@ -174,137 +178,248 @@ The images are tagged with the current Git commit SHA:
-
-Confirm that service types are now available for instantiation:
-
-    kubectl get serviceclasses
+These resources are building blocks of the service catalog in Kubernetes from an
+API standpoint.
 
-This will output available service types, for example:
+----
 
-    NAME               LABELS    DATA
-    booksbe            <none>    {"apiVersion":"...
+#### Note: accessing the service catalog
 
-We can now create instances of these service classes and connect them
-using bindings:
+Unfortunately, `kubectl` doesn't know how to speak to both the service catalog
+API server and the main Kubernetes API server without switching contexts or
+`kubeconfig` files.  For now, the best way to access the service catalog API
+server is via a dedicated `kubeconfig` file.  You can manage the kubeconfig in
+use within a directory using the `direnv` tool.
 
-    # Create backend (MySQL) instance.
-    kubectl create -f backend.yaml
+----
 
-    # Create binding called 'database'.
-    kubectl create -f binding.yaml
+Because we haven't created any resources in the service-catalog API server yet,
+`kubectl get` will return an lists of resources:
 
-Creating the binding will create a `Secret` for binding consumption. This
-can now be used by a native Kubernetes app. You can inspect the config map
-using `kubectl get secret database`:
-
-    NAME       DATA      AGE
-    database   4         55s
-
-Now you can deploy the application that consumes the binding:
-
-    kubectl create -f ../user-bookstore-client/bookstore.yaml
-
-This will create a Kubernetes service `user-bookstore-fe`. Wait for deployments
-to start and an IP address of the frontend to be assigned. You can monitor the
-external IP address creation using `kubectl get services`:
-
-    NAME                    CLUSTER-IP       EXTERNAL-IP       PORT(S)    AGE
-    cf-i-3a121d22-booksbe   10.107.254.221   <none>            3306/TCP   2m
-    user-bookstore-fe       10.107.254.221   **<pending>**         3306/TCP   2m
-    kubernetes              10.107.240.1     <none>            443/TCP    1d
-
-Once the IP address becomes available we can use it to contact the frontend
-endpoint. In this example, the IP address is `104.154.153.120`.
-
-Save the IP address in an environment variable:
-
-    IP=104.154.153.120
-
-And interact with the Bookstore API:
+    kubectl get brokers,serviceclasses,instances,bindings
 
-    # List shelves
-    curl "http://${IP}:8080/shelves"
+### Registering a Broker
 
-    # List a specific shelf
-    curl "http://${IP}:8080/shelves/1"
+First, we'll register a service broker with the catalog.  To do this, we'll
+create a new [`Broker`](../contrib/examples/walkthrough/ups-broker.yaml)
+resource:
 
-    # Create a new shelf
-    curl -H 'Content-Type: application/json' \
-         -H 'x-api-key: 123' \
-          -d '{ "theme": "Travel" }' \
-          "http://${IP}:8080/shelves"
+    kubectl create -f contrib/examples/walkthrough/ups-broker.yaml
```

Comment:
Minor thing but can you add a prompt (e.g. `$`) before each command that the user types so they know what they should do vs what they'll see as output?